langlib/jballerina.java/src/main/ballerina/ballerina_values_to_Java.bal

```diff
@@ -113,7 +113,7 @@ isolated function getStringValue(int v) returns string = @Method {
 # + value - The `JObject` instance which is to be casted
 # + castType - The `JObject` implementation type ``typedesc<JObject>`` to which the given object is casted to if assignable
 # + return - The `JObject|error`, which refers to the new `JObject` instance or an `error`
```

Comment:
We should probably update these too, right? Even the example?

Reply:
Done